components/accelerator/nvidia/query/nvml/testutil/options.go

```diff
@@ -0,0 +1,55 @@
+package testutil
+
+type Op struct {
```

Comment:
Actually we don't need options here? we can just export these fields in mockDevice..

Reply:
removed